<launch>
  <arg name="command_topic" default="/picarx/drive/command" />
  <arg name="motor_left_topic" default="motor_left/speed" />
  <arg name="motor_right_topic" default="motor_right/speed" />
  <arg name="steering_topic" default="/picarx/steer" />

  <arg name="name" default="AckermannSkill" />
  <arg name="uid" default="ackermanntest" />

  <node name="ackermann_skill" pkg="picarx_ackermann_drive" type="ackermann_drive_skill.py" respawn="false" output="screen" args="$(arg name) $(arg uid)">
    <param name="command_topic" type="str" value="$(arg command_topic)" />
    <param name="motor_left_topic" type="str" value="$(arg motor_left_topic)" />
    <param name="motor_right_topic" type="str" value="$(arg motor_right_topic)" />
    <param name="steering_topic" type="str" value="$(arg steering_topic)" />
  </node>

</launch>